src/webgl/GeometryBuilder.js

```diff
@@ -9,6 +9,7 @@ import * as constants from '../core/constants';
 class GeometryBuilder {
   constructor(renderer) {
     this.renderer = renderer;
+    renderer._doStroke = true;
```

Comment:
Btw does this reset back to its previous value after the end of `buildGeometry`? Do we have to put it after the `renderer._pInst.push()` so that it gets automatically reset again after?

Reply:
yup sorry for that!! i'll change it right away!